src/contract/assignments.rs

```diff
@@ -150,6 +153,28 @@ impl ToMerkleSource for OwnedRightsInner {
     }
 }
 
+// Question: Should this also be implemented for ParentOwnedRights?
```

Comment:
I doubt that `ParentOwnedRights` has anything which is currently concealed - but this may change with RGBv4, so better to implement `MergeRevealed` to _all_ types, even blank ones (the impl will just do nothing, but still will be called from the parent structure).

Reply:
Noted. Adding a todo so that we don't forget about it.